lib/suspenders/app_builder.rb

```diff
@@ -54,6 +54,10 @@ def configure_newrelic
       template 'newrelic.yml.erb', 'config/newrelic.yml'
     end
 
+    def configure_honeybadger
+      copy_file 'honeybadger.rb', 'config/initializers/honeybadger.rb'
```

Comment:
I believe the general guideline is to convert the code we touch/change to the current style guide.